src/List/HelpLink.component.js

```diff
@@ -18,8 +23,25 @@ function getDocsVersion({ major, minor, snapshot }) {
     return `${major}.${minor}`;
 }
 
+/** 
```

Comment:
Nice, thanks for adding some docs!

The high-level description of what the function accomplishes is missing. Something like the one on L43-44 is great and tells me a lot of what I can use the function for.